csr/csr.go

```diff
@@ -49,7 +49,7 @@ type BasicKeyRequest struct {
 
 // NewBasicKeyRequest returns a default BasicKeyRequest.
 func NewBasicKeyRequest() *BasicKeyRequest {
-	return &BasicKeyRequest{"ecdsa", curveP256}
+	return &BasicKeyRequest{"rsa", 2048}
```

Comment:
@kisom That should be mentioned somehow. Don't know where exactly, but it should be accessible in cfssl binary.

As for the "hosts" and "hostname" validation, can you suggest one-liner? Or should I use simple loop?